torchgeo/datamodules/l7irish.py

```diff
@@ -48,6 +48,16 @@ def __init__(
             **kwargs,
         )
 
+        self.train_aug = AugmentationSequential(
+            K.Normalize(mean=self.mean, std=self.std),
+            K.RandomResizedCrop(
+                _to_tuple(self.patch_size), scale=(0.6, 1.0), cropping_mode="resample"
```

Comment:
See https://github.com/kornia/kornia/issues/2396. The only way I found to get around this is to use `cropping_mode="resample"`. Apparently it's slower but better?